gulpfile.js

```diff
@@ -0,0 +1,870 @@
+
+var packageJson = require('./package.json');
+
+//Gulp doesn't seem to have a way to get the currently running tasks for setting
+//per-task variables.  We use the command line argument here to detect which task is being run.
+var noDevelopmentGallery = process.argv[2] === 'release' || process.argv[2] === 'makeZipFile';
+var copyUnminified = process.argv[2] === 'combine' || process.argv[2] === 'default' || process.argv[2] === undefined;
+
+var version = packageJson.version;
+if (/\.0$/.test(version)) {
+    version = version.substring(0, version.length - 2);
+}
+
+var sourceFiles = ['Source/**/*.js',
+                   '!Source/*.js',
+                   '!Source/Workers/**',
+                   '!Source/ThirdParty/Workers/**',
+                   'Source/Workers/createTaskProcessorWorker.js'];
+
+var buildFiles = ['Specs/**/*.js',
+                  '!Specs/SpecList.js',
+                  'Source/Shaders/**/*.glsl'];
+
+var jsHintFiles = ['Source/**/*.js',
+                   '!Source/Shaders/**',
+                   '!Source/ThirdParty/**',
+                   '!Source/Workers/cesiumWorkerBootstrapper.js',
+                   'Apps/**/*.js',
+                   'Apps/Sandcastle/gallery/*.html',
+                   '!Apps/Sandcastle/ThirdParty/**',
+                   'Specs/**/*.js',
+                   'Tools/buildTasks/**/*.js',
+                   'gulpfile.js'];
+
+var filesToClean = ['Source/Cesium.js',
+                    'Build',
+                    'Instrumented',
+                    'Source/Shaders/**.js',
+                    'Specs/SpecList.js',
+                    'Apps/Sandcastle/.jshintrc',
+                    'Apps/Sandcastle/jsHintOptions.js',
+                    'Apps/Sandcastle/gallery/gallery-index.js',
+                    'Cesium-*.zip'];
+
+var filesToSortRequires = ['Source/**/*.js',
+                           '!Source/Shaders/**',
+                           '!Source/ThirdParty/**',
+                           '!Source/Workers/cesiumWorkerBootstrapper.js',
+                           '!Source/copyrightHeader.js',
+                           '!Source/Workers/transferTypedArrayTest.js',
+                           'Apps/**/*.js',
+                           '!Apps/Sandcastle/ThirdParty/**',
+                           '!Apps/Sandcastle/jsHintOptions.js',
+                           'Specs/**/*.js',
+                           '!Specs/spec-main.js',
+                           '!Specs/SpecRunner.js',
+                           '!Specs/SpecList.js',
+                           '!Apps/Sandcastle/Sandcastle-client.js',
+                           '!Apps/Sandcastle/Sandcastle-header.js',
+                           '!Apps/Sandcastle/Sandcastle-warn.js',
+                           '!Apps/Sandcastle/gallery/gallery-index.js'];
+
+gulp.task('default', ['combine']);
+
+gulp.task('build', function(done) {
+    mkdirp.sync('Build');
+    glslToJavaScript(false, 'Build/minifyShaders.state');
```

Comment:
You are always passing in false for shader minification.  You can see in the output that all the shader comments are left, even in the minified release build.